docs/user-guide/llamaedge-lobechat.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 5
+---
+
+# LobeChat + LlamaEdge
+
+The LobeChat framework supports OpenAI providers, and since LlamaEdge is compatible with the OpenAI API, you can use LlamaEdge as the backend Large Language Model (LLM) API. This integration supports:
+
+* The hosted LobeChat service
+* Any product built on the open-source LobeChat framework
+
+## Prerequisites
+
+Before proceeding, follow the [Getting started with LlamaEdge guide](get-started-with-llamaedge.md) to run an open-source LLM locally.
```

Comment:
I think we will have a different article for introduction to the API server use cases. But we can change that later.